graphene_sqlalchemy/tests/test_schema.py

```diff
@@ -1,50 +0,0 @@
-from py.test import raises
```

Comment:
FYI I moved those tests to test_types.py because it was not clear where to add new tests.